lib/server/createHttpServer.ts

```diff
@@ -51,7 +51,7 @@ export const createHttpServer = async (port = 3020) => {
     }
 
     if (url.pathname === "/") {
-      const html = await getIndex()
+      const html = await getIndex(circuitName)
```

Comment:
this is a bad positional parameter, if the parameter isn't obvious make it a named object
```suggestion
      const html = await getIndex({ circuitName })
```